crm/crm/components/crm.py

```diff
@@ -52,7 +52,7 @@ def add_contact(self):
                 )
             )
             sess.commit()
-            self.toggle()
+            self.toggle(self)
```

Comment:
This one seems kinda weird.

When I was poking around with a different issue last night, I noticed that I couldn't call another function in the class without explicitly passing `self`. It wasn't like this before, and I'm not sure if we want to just hunker down and accept this.

This is going to trip people up.

@picklelo what do you think? do you know why `self` has to be passed now?

Reply:
> This one seems kinda weird.
> 
> When I was poking around with a different issue last night, I noticed that I couldn't call another function in the class without explicitly passing `self`. It wasn't like this before, and I'm not sure if we want to just hunker down and accept this.
> 
> This is going to trip people up.
> 
> @picklelo what do you think? do you know why `self` has to be passed now?

 I don't know why we need to add self in version 0.1.27 either. 
The code to return async function also have some changes in its behavior. I also fixed it in traversal example too.
If we don't add self , it will be failed in pynecone 0.1.27. 
```
Traceback (most recent call last):
  File "/Users/pebblarmbp/anaconda3/envs/pynecone-examples/lib/python3.11/site-packages/pynecone/state.py", line 655, in _process_event
    events = fn(**payload)
             ^^^^^^^^^^^^^
  File "/Users/pebblarmbp/git/py-webapp/crm/crm/components/crm.py", line 57, in add_contact
    self.toggle()
TypeError: AddModalState.toggle() missing 1 required positional argument: 'self'
```


But Pynecone 0.1.27 has already been released on PyPI and all developers can install through `pip install pynecone`
The next step for them is to clone the pynecone-examples source code to test. 
If they get the wrong example code., they will feel strange.

Reply:
@milochen0418 the issue already has a partial fix in main that hasn't been released https://github.com/pynecone-io/pynecone/issues/896

I don't think having to pass self was the intended behavior, although the fix regresses other areas as well (which I have filed issues for in the main pynecone and pcweb repos)

Reply:
@masenf 
Hi Thanks ~
What thing should we release the example for pynecone 0.1.27?
Do I need to cancel all the returned functions that send self?

Reply:
I will change it back as soon as the next version of pynecone is released.

Reply:
Awesome thanks! We should have a new version out tonight hopefully